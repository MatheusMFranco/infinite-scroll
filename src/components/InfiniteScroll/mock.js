export default class {
  constructor() {
    this.root = null;
    this.rootMargin = '';
    this.thresholds = [];
  }

  disconnect() {}

  observe() {}

  takeRecords() {
    return [];
  }

  unobserve() {}
}
