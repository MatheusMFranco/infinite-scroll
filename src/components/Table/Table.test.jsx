import { screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';

import { Table } from '.';
import { renderTheme } from '../../utils/render-theme';

import mock from './mock.js';

describe('<Table />', () => {
  it('should render lines', () => {
    expect.assertions(1);
    renderTheme(<Table data={mock.data} />);
    expect(screen.getAllByRole('row')).toHaveLength(mock.data.length);
  });

  it('should active line', () => {
    expect.assertions(1);
    renderTheme(<Table data={mock.data} />);
    const line = screen.getAllByRole('row')[2];
    userEvent.click(line);
    expect(line).toHaveAttribute('class', 'active');
  });

  it('should activate just one line', () => {
    expect.assertions(3);
    renderTheme(<Table data={mock.data} />);
    const line = screen.getAllByRole('row')[0];
    userEvent.click(line);
    expect(line).toHaveAttribute('class', 'active');
    const secondLine = screen.getAllByRole('row')[50];
    userEvent.click(secondLine);
    expect(line).not.toHaveAttribute('class', 'active');
    expect(secondLine).toHaveAttribute('class', 'active');
  });

  it('should render four cells', () => {
    expect.assertions(5);
    const { container } = renderTheme(<Table data={mock.data} />);
    const row = container.firstChild
      .querySelectorAll('tr')[0]
      .querySelectorAll('td');
    expect(row).toHaveLength(4);
    expect(row[0]).toContainElement(screen.getAllByRole('img')[0]);
    expect(row[1].textContent).toBe('e906859036f847e68e58a577da9312a2');
    expect(row[2].textContent).toBe('item-0');
    expect(+row[3].textContent).toBe(0);
  });

  it('should match snapshot', () => {
    const { container } = renderTheme(<Table data={mock.data} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
