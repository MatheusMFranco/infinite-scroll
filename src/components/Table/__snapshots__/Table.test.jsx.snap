// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Table /> should match snapshot 1`] = `
.c0 {
  border: 0.1rem solid #000;
  border-collapse: collapse;
  width: 100%;
}

.c0 tr {
  border: 0.1rem solid #000;
  -webkit-transition: all 500ms ease-in-out;
  transition: all 500ms ease-in-out;
}

.c0 tr:hover {
  cursor: pointer;
  background-color: rgba(0,0,0,0.4);
}

.c0 tr td {
  padding: 0 1rem;
}

.c0 tr td:first-child,
.c0 tr td:last-child {
  width: 3rem;
}

.c0 tr td img {
  padding-top: 0.3rem;
  color: #000;
}

.c0 .active {
  background-color: rgba(0,0,0,0.4);
}

<table
  class="c0"
>
  <tbody>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        e906859036f847e68e58a577da9312a2
      </td>
      <td>
        item-0
      </td>
      <td>
        0
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        761ff52b8e6dd373fdf291a1a70df20c
      </td>
      <td>
        item-1
      </td>
      <td>
        1
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        38d3f385ea8d8bb2dcfc759ac85af6ef
      </td>
      <td>
        item-2
      </td>
      <td>
        2
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        65910a0fd76f69a08e6faa142384f327
      </td>
      <td>
        item-3
      </td>
      <td>
        3
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        05e4075a3f97ed7cb0bd40329caa2423
      </td>
      <td>
        item-4
      </td>
      <td>
        4
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        ad36abff56ce76a479dda134a3744c3b
      </td>
      <td>
        item-5
      </td>
      <td>
        5
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        7421b1b797e5710c7ee3726a1f6e6a8b
      </td>
      <td>
        item-6
      </td>
      <td>
        6
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        3a4ef5b78524c0580352bcdc969922cd
      </td>
      <td>
        item-7
      </td>
      <td>
        7
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        226337775a640fd25fbd2f0ca9b4fa7e
      </td>
      <td>
        item-8
      </td>
      <td>
        8
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        d1b07b282926f1293feb7c4af47a7c95
      </td>
      <td>
        item-9
      </td>
      <td>
        9
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        394f3c2800035af92506ec7c5d7f755c
      </td>
      <td>
        item-10
      </td>
      <td>
        10
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        ab42d325bbd870cfcd706d73e6c9946f
      </td>
      <td>
        item-11
      </td>
      <td>
        11
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        6d1f0d6d154c71959b3015e5f5165fc1
      </td>
      <td>
        item-12
      </td>
      <td>
        12
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        a51f63e7d81b293c6f66f6eaa1415838
      </td>
      <td>
        item-13
      </td>
      <td>
        13
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        c8b9b9a564aef7e9874edde58a44abfa
      </td>
      <td>
        item-14
      </td>
      <td>
        14
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        b43e9e67163ba6beddb37ef9e77ea7cb
      </td>
      <td>
        item-15
      </td>
      <td>
        15
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        941cf308e6b24640195900263dbea458
      </td>
      <td>
        item-16
      </td>
      <td>
        16
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        7faf4030623085f256bb139c308917bc
      </td>
      <td>
        item-17
      </td>
      <td>
        17
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        250c0eeeb64242dfaf6692140b6fb76f
      </td>
      <td>
        item-18
      </td>
      <td>
        18
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        e3aca5b7a1478bd4f63caf03251a1fd2
      </td>
      <td>
        item-19
      </td>
      <td>
        19
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        5cdbd9d6a64bd1cfc2cd65fb14ed8efa
      </td>
      <td>
        item-20
      </td>
      <td>
        20
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        535aceab0143f7e1cd629caedef3e0ea
      </td>
      <td>
        item-21
      </td>
      <td>
        21
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        b3b4475b7f1a4bba743c0ca229c89b90
      </td>
      <td>
        item-22
      </td>
      <td>
        22
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        38d555cee07a3207d99328f7fae8e046
      </td>
      <td>
        item-23
      </td>
      <td>
        23
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        1092543d121a53714823a50491ca7ebc
      </td>
      <td>
        item-24
      </td>
      <td>
        24
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        aa7f9f8332803e2f3d1cd5df46cd70b1
      </td>
      <td>
        item-25
      </td>
      <td>
        25
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        2fc098759adeb42aeffc0d0f37cc5395
      </td>
      <td>
        item-26
      </td>
      <td>
        26
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        c98ab7bd4bfe916707c347313722a854
      </td>
      <td>
        item-27
      </td>
      <td>
        27
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        266a3c68d87071a9b993456df1beb9ce
      </td>
      <td>
        item-28
      </td>
      <td>
        28
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        dd7c8cf404981c52e644bdb4f806c1d2
      </td>
      <td>
        item-29
      </td>
      <td>
        29
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        e865fb4953f98803b42dc8a62e7709d7
      </td>
      <td>
        item-30
      </td>
      <td>
        30
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        75c54c54313aaec82be42bf048db9ae4
      </td>
      <td>
        item-31
      </td>
      <td>
        31
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        f89f7fe2248c07d990eb0cf2ab31162f
      </td>
      <td>
        item-32
      </td>
      <td>
        32
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        85aa8182f24277c397609e0ef4933958
      </td>
      <td>
        item-33
      </td>
      <td>
        33
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        19c03b01a6bd64b1d1761b5defbff882
      </td>
      <td>
        item-34
      </td>
      <td>
        34
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        05d48faa4c919b28fe13b4af47250291
      </td>
      <td>
        item-35
      </td>
      <td>
        35
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        347fd3e817e62c79efd5da1d6e2f821b
      </td>
      <td>
        item-36
      </td>
      <td>
        36
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        01d1df7d5f6dfdbc39b85a9455231f6c
      </td>
      <td>
        item-37
      </td>
      <td>
        37
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        65ecc7df7d65b07e6dfd84dbbd9a0d17
      </td>
      <td>
        item-38
      </td>
      <td>
        38
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        d11b33370f73ca05eb6fd1e313531062
      </td>
      <td>
        item-39
      </td>
      <td>
        39
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        9f2447525a856fce30ba34da2438af27
      </td>
      <td>
        item-40
      </td>
      <td>
        40
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        7c22bdd5f22d18737cef995129bc113c
      </td>
      <td>
        item-41
      </td>
      <td>
        41
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        0fc0f07e09a7d396d92ee9c6875153f7
      </td>
      <td>
        item-42
      </td>
      <td>
        42
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        d55cc93e761e14bcbb13fdc7cffad88b
      </td>
      <td>
        item-43
      </td>
      <td>
        43
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        a6c02ec95155ed66a99d7fb1760a1330
      </td>
      <td>
        item-44
      </td>
      <td>
        44
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        a9d398a9e2477797f0fb989a8de7d4c2
      </td>
      <td>
        item-45
      </td>
      <td>
        45
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        7cfd60059b7c629f6a1359a5fcea4d33
      </td>
      <td>
        item-46
      </td>
      <td>
        46
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        51eef11c0a3ccbe29d4ab0a7f53dad9b
      </td>
      <td>
        item-47
      </td>
      <td>
        47
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        c6b29df3b23434b0c1ff7de887bfb230
      </td>
      <td>
        item-48
      </td>
      <td>
        48
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        2b4d92ee58d1239a140a47f6a7f75de8
      </td>
      <td>
        item-49
      </td>
      <td>
        49
      </td>
    </tr>
    <tr
      class="active"
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        7bdb7ab6e7f3d4f676f7e5ea920e9d70
      </td>
      <td>
        item-50
      </td>
      <td>
        50
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        db178c56fd988d6fa32bca451fcf6d90
      </td>
      <td>
        item-51
      </td>
      <td>
        51
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        b9eb4e6974ae35e9050a7536f4b296b1
      </td>
      <td>
        item-52
      </td>
      <td>
        52
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        511dc2dc8b5896b17da20f8ce3ba9b97
      </td>
      <td>
        item-53
      </td>
      <td>
        53
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        21ba555fdf6ea2e64ee1c887791a4e0e
      </td>
      <td>
        item-54
      </td>
      <td>
        54
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        aec377296dce715b3e5b6a33223a35f0
      </td>
      <td>
        item-55
      </td>
      <td>
        55
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        e06e5fec86765f4618bd9bb7d36d7523
      </td>
      <td>
        item-56
      </td>
      <td>
        56
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        e2d4c9dc1ec7539f3127dfc57a0adfdf
      </td>
      <td>
        item-57
      </td>
      <td>
        57
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        a7f046199534000b4054636e15052f6c
      </td>
      <td>
        item-58
      </td>
      <td>
        58
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        31f06570355be090b10b6a3451676e76
      </td>
      <td>
        item-59
      </td>
      <td>
        59
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        5393c2d06bef23521c861492cbb353ae
      </td>
      <td>
        item-60
      </td>
      <td>
        60
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        e32c707f9f1d7d8764a9879f40119b4e
      </td>
      <td>
        item-61
      </td>
      <td>
        61
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        3dcc411f0a3fb9928928be20187503a5
      </td>
      <td>
        item-62
      </td>
      <td>
        62
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        7d9a0819675274d487199e4a7107bdb2
      </td>
      <td>
        item-63
      </td>
      <td>
        63
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        5ec9f0a960af5a7acd27bb53e7d26a63
      </td>
      <td>
        item-64
      </td>
      <td>
        64
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        1ead171c46f482339e281292547bfa84
      </td>
      <td>
        item-65
      </td>
      <td>
        65
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        05c498bb606a718fbcb36d785e1ec291
      </td>
      <td>
        item-66
      </td>
      <td>
        66
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        4800f79743bfcb51c4ea1f42257f8ba3
      </td>
      <td>
        item-67
      </td>
      <td>
        67
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        96a7ba878bfeef87d03e05c111c0d12b
      </td>
      <td>
        item-68
      </td>
      <td>
        68
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        89386667437bea2f679a2186cdd4e7bb
      </td>
      <td>
        item-69
      </td>
      <td>
        69
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        95d0cdfddadac847a717c0ff38cd4ce4
      </td>
      <td>
        item-70
      </td>
      <td>
        70
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        ce7bfc635c9f7344561910624638c31f
      </td>
      <td>
        item-71
      </td>
      <td>
        71
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        d724e32235073c9a0072c8ba63004be9
      </td>
      <td>
        item-72
      </td>
      <td>
        72
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        28d503b70c0384696501c81b8b51319c
      </td>
      <td>
        item-73
      </td>
      <td>
        73
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        e26b3a73f66a16c9d0658f31f3730e75
      </td>
      <td>
        item-74
      </td>
      <td>
        74
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        c64156c2ae35edc0e6cf9fed350634ca
      </td>
      <td>
        item-75
      </td>
      <td>
        75
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        375771667dfabe97ee457fa08a091ba2
      </td>
      <td>
        item-76
      </td>
      <td>
        76
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        57f42b26ab7da8e9ed5c7211575de6b5
      </td>
      <td>
        item-77
      </td>
      <td>
        77
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        a695d04fbf1f4dd9bc26dd8d30881c01
      </td>
      <td>
        item-78
      </td>
      <td>
        78
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        4463f8b9ce4c352c84a7d9fcfa914161
      </td>
      <td>
        item-79
      </td>
      <td>
        79
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        0b5b796c67655bad7bbbe2159ef98ccd
      </td>
      <td>
        item-80
      </td>
      <td>
        80
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        b77f0ecc1c06036f69adbfd876d71cc4
      </td>
      <td>
        item-81
      </td>
      <td>
        81
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        9b9ca4295a3a08bbf80e6e39be0efd6a
      </td>
      <td>
        item-82
      </td>
      <td>
        82
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        f84ff74f9aa65450fd7662eeaef3f124
      </td>
      <td>
        item-83
      </td>
      <td>
        83
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        835de0286c96015eb30df333cd4599b8
      </td>
      <td>
        item-84
      </td>
      <td>
        84
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        aed9bd38dc12df8fa89d93943f8861bb
      </td>
      <td>
        item-85
      </td>
      <td>
        85
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        3234479528d9a18432d400797807ccd1
      </td>
      <td>
        item-86
      </td>
      <td>
        86
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        c1d9b0b0c15e6ed8842847d8dd161a10
      </td>
      <td>
        item-87
      </td>
      <td>
        87
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        32ebfe8f16fa3e1f053f64f01340c48a
      </td>
      <td>
        item-88
      </td>
      <td>
        88
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        c0fe70233b82c736210f969b6f80be8c
      </td>
      <td>
        item-89
      </td>
      <td>
        89
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        94d1f02970f4287c738244c7265121c5
      </td>
      <td>
        item-90
      </td>
      <td>
        90
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        da6a43255354c352b2bcae58a3e1aee8
      </td>
      <td>
        item-91
      </td>
      <td>
        91
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        1a6066becde7b346ec4382624feb5fa6
      </td>
      <td>
        item-92
      </td>
      <td>
        92
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        a3a5f60fdd5b347ac102d7c2e7bfb151
      </td>
      <td>
        item-93
      </td>
      <td>
        93
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        ed35c3723e7ffef36e11b3f19e089115
      </td>
      <td>
        item-94
      </td>
      <td>
        94
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        5b9a202460659923e82375fc9c1c7ba5
      </td>
      <td>
        item-95
      </td>
      <td>
        95
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        17452f0cbd3ffe31674b5b112484e209
      </td>
      <td>
        item-96
      </td>
      <td>
        96
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        7caf4d76e506362e2217a7b4687e0f6a
      </td>
      <td>
        item-97
      </td>
      <td>
        97
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        1c10ad4f94c2d0423de76d1f0ed6645a
      </td>
      <td>
        item-98
      </td>
      <td>
        98
      </td>
    </tr>
    <tr
      class=""
    >
      <td>
        <img
          alt="📁"
          aria-label="icon"
          src="assets/icons/file.svg"
        />
      </td>
      <td>
        d53ceea67d7c6d909db60f98085facbd
      </td>
      <td>
        item-99
      </td>
      <td>
        99
      </td>
    </tr>
  </tbody>
</table>
`;
